addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -154,6 +154,19 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         FontFamilyPickerUserValueWidget.prototype.fontVariables = fontVariables;
         return this._super(...arguments);
     },
+    /**
+     * @override
+     */
+    _patchForComputeSnippetTemplates($html) {
+        this._super(...arguments);
+
+        // TODO adapt in master: as a stable fix we corrected the behavior of the
+        // logo button that led to an error when switching from Text to Logo.
+        // Remove me in master
+        let logo_button = $html.find('[data-customize-website-views="website.option_header_brand_logo"]')[0];
+        logo_button.dataset.customizeWebsiteViews = "|website.option_header_brand_name|website.option_header_brand_logo";
+        logo_button.dataset.resetViewArch = "true";
```

Comment:
@SergeBayet You should add something like this here
```js
const headerBrandSelectorEl = $html.find('[data-name="option_header_brand_none"]')[0].closest("[data-selector]");
headerBrandSelectorEl.dataset.selector = "#wrapwrap > header nav.navbar .navbar-brand";
```
because the ".logo" class in the `data-selector` attribute added by [this commit](https://github.com/odoo/odoo/commit/5cf32b489f15f5124faa6ea4f09b42de4ab01ee9) prevents the navbar brand options to be visible when "text" logo is enabled... But we need to be able to set the logo height even it is a "Text" logo...
As it was the case in previous versions..

Reply:
My bad, I overlooked it.

Reply:
@bvr-odoo